charts/karpenter/values.yaml

```diff
@@ -74,7 +74,9 @@ topologySpreadConstraints:
     topologyKey: topology.kubernetes.io/zone
     whenUnsatisfiable: ScheduleAnyway
 # -- Tolerations to allow the pod to be scheduled to nodes with taints.
-tolerations: []
+tolerations:
```

Comment:
we should call this out in the release notes, or maybe upgrade guide. I agree that it should probably tolerate this. But might be surprising if Karpenter reschedules on an upgrade because someone wasn't tolerating this before.